docs-parts/computation/01-autopopulate_lang4.rst

```diff
@@ -0,0 +1,27 @@
+The function ``parpopulate`` works identically to ``populate`` except that it uses a job reservation mechanism to allow multiple processes to populate the same table in parallel without collision.
+To enable ``parpopulate``, create the job reservation table ``<package>.Jobs`` which must have the following declaration:
+
+.. code-block:: matlab
+
+  %  <package>.Jobs (job) # the job reservation table
```

Comment:
This first line is no longer used in table definition.  This was from old syntax. Now it should just start with the comment:
```suggestion
  # Job reservation table
```